src/commands/lambda/prompt.ts

```diff
@@ -0,0 +1,195 @@
+      if (!value || !sentenceMatchesRegEx(value, AWS_SECRET_ACCESS_KEY_REG_EXP)) {
+        return 'Enter a valid AWS Secret Access Key.'
+      }
+
+      return true
+    },
+  },
+  {
+    // AWS_DEFAULT_REGION
+    choices: AWS_REGIONS,
+    default: 0,
+    message: 'Select an AWS region:',
+    name: AWS_DEFAULT_REGION_ENV_VAR,
+    type: 'list',
+  },
+]
+
+export const datadogApiKeyTypeQuestion = (datadogSite: string): ListQuestion => ({
+  choices: [
+    {
+      name: `Plain text ${bold('API Key')} (Recommended for trial users) `,
+      value: {
+        envVar: CI_API_KEY_ENV_VAR,
+        message: 'API Key:',
+      },
+    },
+    new Separator(),
+    {
+      name: `API key encrypted with AWS Key Management Service ${bold('(KMS) API Key')}`,
+      value: {
+        envVar: CI_KMS_API_KEY_ENV_VAR,
+        message: 'KMS Encrypted API Key:',
+      },
+    },
+    {
+      name: `AWS Secrets Manager ${bold('API Key Secret ARN')}`,
+      value: {
+        envVar: CI_API_KEY_SECRET_ARN_ENV_VAR,
+        message: 'API Key Secret ARN:',
+      },
+    },
+  ],
+  message: `Which type of Datadog API Key you want to set? \nLearn more at ${blueBright(
+    `https://app.${datadogSite}/organization-settings/api-keys`
+  )}`,
+  name: 'type',
+  type: 'list',
+})
+
+const datadogSiteQuestion: ListQuestion = {
+  // DATADOG SITE
+  choices: SITES,
+  message: `Select the Datadog site to send data. \nLearn more at ${blueBright(
+    'https://docs.datadoghq.com/getting_started/site/'
+  )}`,
+  name: CI_SITE_ENV_VAR,
+  type: 'list',
+}
+
+export const datadogEnvVarsQuestions = (datadogApiKeyType: Record<string, any>): InputQuestion => ({
+  // DATADOG API KEY given type
+  default: process.env[datadogApiKeyType.envVar],
+  message: datadogApiKeyType.message,
+  name: datadogApiKeyType.envVar,
+  type: 'input',
+  validate: (value) => {
+    if (!value || !sentenceMatchesRegEx(value, DATADOG_API_KEY_REG_EXP)) {
+      return 'Enter a valid Datadog API Key.'
+    }
+
+    return true
+  },
+})
+
+export const confirmationQuestion = (message: string): ConfirmQuestion => ({
+  message,
+  name: 'confirmation',
+  type: 'confirm',
+})
+
+export const functionSelectionQuestion = (functionNames: string[]): CheckboxQuestion => ({
+  choices: functionNames,
+  message: 'Select the functions to modify',
+  name: 'functions',
+  type: 'checkbox',
+  validate: (selectedFunctions) => {
+    if (selectedFunctions.length < 1) {
+      return 'You must choose at least one function.'
+    }
+
+    return true
+  },
+})
+
+export const requestAWSCredentials = async () => {
+  try {
+    const awsCredentialsAnswers = await prompt(awsCredentialsQuestions)
+    process.env[AWS_ACCESS_KEY_ID_ENV_VAR] = awsCredentialsAnswers[AWS_ACCESS_KEY_ID_ENV_VAR]
+    process.env[AWS_SECRET_ACCESS_KEY_ENV_VAR] = awsCredentialsAnswers[AWS_SECRET_ACCESS_KEY_ENV_VAR]
+    process.env[AWS_DEFAULT_REGION_ENV_VAR] = awsCredentialsAnswers[AWS_DEFAULT_REGION_ENV_VAR]
```

Comment:
Let's also ask for `AWS_SESSION_TOKEN` as an **optional** env var, which is needed when using temporary credentials (fairly common) https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp_use-resources.html#using-temp-creds-sdk-cli

Reply:
Added it!